wger/manager/models.py

```diff
@@ -643,6 +643,18 @@ class Setting(models.Model):
     """
     The weight unit of a set. This can be e.g. kg, lb, km/h, etc.
     """
+    NUMBERS = [(1, 1), (2, 2), (3, 3), (4, 4), (5, 5), (6, 6), (7, 7), (8, 8), (9, 9), (10, 10)]
+    rpe = models.IntegerField(verbose_name=_('rpe'),
+                              default=5,
+                              blank=True,
+                              null=True,
+                              choices=NUMBERS)
+
```

Comment:
Can you change this to rir (Reps In Reserve) instead? You can leave the options 4 to 1 in 0,5 steps. You can leave the field as Integer and make the second value in the touple a string.